styles/desktop.css

```diff
@@ -0,0 +1,345 @@
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+}
+
+.card-wrapper:nth-child(even) {
+  flex-direction: row-reverse;
+}
+
+.card-content-wrapper {
+  margin: 46px;
+}
+
+.card-wrapper img {
+  width: 544px;
+}
+
+.card-wrapper h3 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.see-projects-btn {
+  margin-top: 40px;
+  padding: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: rgba(64, 83, 252, 1);
+  background: rgba(57, 109, 242, 0);
+  border: 2px solid rgba(64, 83, 252, 1);
+  border-radius: 10px;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  -ms-border-radius: 10px;
+  -o-border-radius: 10px;
+  transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -webkit-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -moz-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -ms-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+  -o-transition: background-color 1s, color 0.5s, rotate 0.8s ease-in-out;
+}
+
+.see-projects-btn:hover {
+  background-color: teal;
+  color: white;
+  rotate: 10deg;
+  border: none;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.tech-used li {
+  background: rgba(235, 235, 255, 1);
+  padding: 4px 12px;
+  color: rgba(96, 112, 255, 1);
+  border-radius: 8px;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+}
+
+.canopy-list {
+  display: flex;
+  gap: 12px;
+}
+
+.card-empty-list {
+  width: 8px;
+  height: 8px;
+  align-self: center;
+  background-color: #c1c7d0;
+  border-radius: 8px;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+}
+
+.canopy-list li:nth-child(3),
+.canopy-list li:nth-child(5) {
+  color: rgba(107, 119, 140, 1);
+}
+
+.card-content-wrapper p {
+  color: rgba(52, 69, 99, 1);
+}
+
+.tech-used {
+  display: flex;
+  gap: 8px;
+}
+
+.last-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 48px;
+  padding: 166px 143px;
+  padding-bottom: 216px;
+  background-color: #fff;
+  border-radius: 0 100px 0 0;
+  -webkit-border-radius: 0 100px 0 0;
+  -moz-border-radius: 0 100px 0 0;
+  -ms-border-radius: 0 100px 0 0;
+  -o-border-radius: 0 100px 0 0;
+}
+
+.last-section h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.last-section p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-social h3 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.language-item,
+.frameworks-item,
+.skills-item,
+.language-list {
+  display: flex;
+  align-items: center;
+}
+
+.language-item,
```

Comment:
- Kindly make sure to use the right color for this list to stick to the Figma design :recycle: 

Your list:
![Screenshot from 2023-04-07 11-26-08](https://user-images.githubusercontent.com/97834160/230584051-46a0a1d8-3285-4ab9-8c5d-064afe00c07e.png)

Figma's:

![Screenshot from 2023-04-07 11-27-57](https://user-images.githubusercontent.com/97834160/230584113-c3215792-b7ca-4d62-9815-357a00fe6386.png)

Reply:
Thank you